codegen/service/security_test.go

```diff
@@ -70,3 +70,31 @@ func TestSecureEndpoint(t *testing.T) {
 		})
 	}
 }
+
+func TestSecureWithSkipRequestBodyEncodeDecode(t *testing.T) {
+	cases := []struct {
+		Name string
+		DSL  func()
+		Code string
+	}{
+		{"with-basicauth", testdata.EndpointWithBasicAuthAndSkipRequestBodyEncodeDecodeDSL, testdata.EndpointWithBasicAuthAndSkipRequestBodyEncodeDecodeCode},
+		//{"with-jwtauth", testdata.EndpointWithJWTAuthAndSkipRequestBodyEncodeDecodeDSL, ""},
```

Comment:
Can you please remove this commented out line? thank you!

Reply:
Done